wherehows-backend/app/msgs/EtlJobMessage.java

```diff
@@ -14,34 +14,25 @@
 package msgs;
 
 import com.fasterxml.jackson.databind.JsonNode;
+import java.util.Properties;
 import metadata.etl.models.EtlType;
-import metadata.etl.models.EtlJobName;
 import metadata.etl.models.RefIdType;
+import wherehows.common.Constant;
 
 
 /**
  * Created by zechen on 9/4/15.
  */
 public class EtlJobMessage {
+  private static int DEFAULT_TIMEOUT = 20000;
+
   private Long whEtlExecId;
```

Comment:
Here: https://github.com/linkedin/WhereHows/pull/542/files#diff-85b5b6498842074eb0404f583e039e36R84

It can't be set until the entry is created in wh_job_execution_history.